src/components/LayoutContainer.tsx

```diff
@@ -17,28 +25,52 @@ const NavBar: React.SFC<{ mobile?: boolean }> = ({
   <div className={`navbar ${mobile ? "mobile" : "desktop"}`}>{children}</div>
 );
 
-const LayoutContainer: React.SFC<propTypes> = ({
-  activePage,
-  children
-}): JSX.Element => {
+const LayoutContainer: React.SFC<{
+  activePath?: Paths | null;
+  sessionUser?: User | null;
+  setSessionUser: (sessionUser: User | null) => void;
+}> = ({ activePath, children, sessionUser, setSessionUser }): JSX.Element => {
   const [sidebarOpen, setSidebarOpen] = useState(false);
-
   function handleSidebarHide() {
     setSidebarOpen(false);
   }
-
   function handleToggle() {
     setSidebarOpen(!sidebarOpen);
   }
 
+  const handleLogout = () => {
+    fetch("/logout", { method: "POST" }).then(response => {
+      if (response.ok) {
```

Comment:
If response isn't okay, is it still safe to set session user to null?

Reply:
Oh, I figured if the request failed it probably didn't actually log the user out